curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md

```diff
@@ -0,0 +1,365 @@
+---
+id: 66a37f37ef5823a313de8c26
+title: Build a Moon Orbit
+challengeType: 14
+dashedName: build-a-moon-orbit
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you will create a simple animation of the Moon's orbit around the Earth using HTML and CSS. The Earth will be at the center of the system, and the Moon will orbit around it.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should align all the elements to the center of the page by setting the height to `100%` of the viewport and setting a flexbox layout in the `body` element.
+   
+2. You should have a `div` with the class `space`.
+
+3. Inside the `.space` `div`, there should be two more `div` elements with the class `earth` and `orbit`, respectively, in order.
+
+4. Inside the `.orbit` `div`, there should be another `div` with the class `moon`.
+
+5. The `space` `div` should use `relative` positioning.
+   
+6. The `space` `div` should be centered on the page and have a width and height of `200px`.
```

Comment:
These two can maybe be deleted? I don't see any tests for them, and I didn't add those properties to my project and it turned out as expected I think.